app/models/ticket.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+class Ticket < ApplicationRecord
+  enum :ticket_type, {
+    edit_person: "edit_person",
+  }
+
+  has_many :ticket_logs
+  has_many :ticket_stakeholders
+  belongs_to :metadata, polymorphic: true
+
+  def user_stakeholders(user)
+    return [] if user.nil?
+    ticket_stakeholders.select do |stakeholder|
+      if stakeholder.stakeholder_type == :user
+        stakeholder.stakeholder_id == user.id
+      elsif stakeholder.stakeholder_type == TicketStakeholder.stakeholder_types[:user_group]
+        user.active_roles.any? { |role| role.group_id == stakeholder.stakeholder_id }
```

Comment:
I think `user.active_roles.where(group_id: stakeholder.stakeholder_id).any?` might be more efficient.

Reply:
Oh yes, thanks.